scenarioo-client/app/components/detailarea/detailarea.component.ts

```diff
@@ -53,6 +53,20 @@ export class DetailareaComponent {
     @Output('valueChange')
     panelCollapsed: EventEmitter<boolean> = new EventEmitter<boolean>();
 
+    ngOnInit(): void {
+        this.setHeightOfDetailarea();
+    }
+
+    @HostListener('window:resize', ['$event'])
+    onResize() {
+        this.setHeightOfDetailarea();
+    }
+
+    setHeightOfDetailarea() {
```

Comment:
can this be private or is it called in the template as well?

Reply:
it can be private